Backend/server.py

```diff
@@ -36,6 +36,14 @@ def data4():
     data = connector.getFilteredSystems(filters)
     return jsonify(data)
 
+# expects json describing filters, returns the systems that satisfy filters
```

Comment:
@lRolwes I think yo uneed to Change the comments here

Reply:
Fixed, sorry about that